src/Components/Components/src/IPersistentComponentStateStore.cs

```diff
@@ -6,19 +7,19 @@ namespace Microsoft.AspNetCore.Components.Lifetime
     /// <summary>
```

Comment:
The namespace should also change, shouldn't it? Looks like it's still `Microsoft.AspNetCore.Components.Lifetime` here.

Reply:
Updated the namespace